### PR TITLE
Update continuous-integration-faqs.md

### PR DESCRIPTION
**Pull request by hrprz-overops**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by jenniferopal — DISMISSED** [review]
lgtm!

**Review by jenniferopal — APPROVED** [review]
lgtm!

File: kb/continuous-integration/continuous-integration-faqs.md [hunk 1]
```diff
@@ -2250,6 +2250,12 @@ The error `java.lang.IllegalStateException: Failed to execute ApplicationRunner`
 #### How can I a run background step in debug mode in Harness?
 To enable debug logging, set the environment variable to `DEBUG=true`. For more configuration information, go to [Relay proxy configuration reference](https://developer.harness.io/docs/feature-flags/relay-proxy/configuration).
```
Comment (nofarblue) [inline kb/continuous-integration/continuous-integration-faqs.md:2251]: " For more configuration information, go to [Relay proxy configuration reference](https://developer.harness.io/docs/feature-flags/relay-proxy/configuration)."

^ this doesn't seem to be related to the question

**Review by nofarblue — CHANGES_REQUESTED** [review]
Please see my comments. thanks

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**